app/invocation/cache_requests_card.tsx

```diff
@@ -626,6 +693,43 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
         className={
           this.getGroupBy() === cache.GetCacheScoreCardRequest.GroupBy.GROUP_BY_TARGET ? "group-by-target" : ""
         }>
+        {capabilities.config.bazelButtonsEnabled && (
```

Comment:
I'm cool to submit this as-is behind a flag - but before we flip the flag in prod, let's find a location for the button that doesn't take up so much vertical space by itself.

Reply:
Sounds good. Would you prefer if the button is on the same line as the title of the cache requests card? Or open to other suggestions

Reply:
Yeah maybe something like that - it feels more like a secondary action rather than a primary action